modules/ROOT/pages/to-create-an-asset.adoc

```diff
@@ -149,10 +149,40 @@ A GraphQL API schema designed with Apollo GraphQL specific directives and struct
 To create a GraphQL API:
 
 . In Exchange, select *Publish new asset*.
-. Enter the asset portal name.
-. From the drop-down list, select the asset type *GraphQL API* .
+. Enter a name for the asset.
+. From the drop-down list, select the asset type *GraphQL API*.
 . Select which GraphQL API schema to upload, GraphQL API schema or GraphQL Apollo API schema.
-. To select the file to upload, click *Choose the file* .
+. To select the file to upload, click *Choose file*.
+. To edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
+. Select an option for *Lifecycle state*. The default is *Stable*.
+. Select *Publish*.
+
+== Create an RPA Activity Template Asset
+Activity templates are reusable code within RPA that standardize the most used activity steps and make them available to other developers in the organization. These templates are .cawl files that are imported in the RPA Activity library for developers to use as part of their RPA processes.
+
+To create an RPA activity template asset:
+
+. In Exchange, select *Publish new asset*.
+. Enter a name for the asset.
+. From the drop-down list, select the asset type *RPA Activity Template*.
+. To select the .cawl file to upload, click *Choose file*.
+. To edit the advanced settings, *GroupId*, *AssetId*, and *Version* (GAV), click *Advanced*. 
++
+Exchange generates the group ID, asset ID, and version for you, and you can change these values as needed. Use the advanced settings to change the asset version.
+. Select an option for *Lifecycle state*. The default is *Stable*.
+. Select *Publish*.
+
+== Create an RPA Process Template Asset
+Templates for RPA processes are imported into RPA Builder as projects and can be further customized before deploying to RPA bots. Process templates are built on common patterns for RPA to build automation faster. These templates are .crpa files that are exported from RPA Builder to share with other developers.
```

Comment:
Any way to avoid passive voice here?

Reply:
Does this work?
Templates for RPA processes are imported into RPA Builder as projects and are customizable before deploying to RPA bots.

Reply:
Rewrote the sentence to be active...I think.